### PR TITLE
[TextFields] Updates to controller header.

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: components/TextFields/src/MDCTextInputController.h [hunk 1]
```diff
@@ -82,7 +82,7 @@
 /**
  Default value for disabledColor.
 
- Default is blue.
+ Default is [UIColor lightGrayColor].
```
Comment (romoore) [inline components/TextFields/src/MDCTextInputController.h]: Super-duper-nit: `lightGrayColor` is a property. Prefer either

    Default is .lightGrayColor.

or

    Default is UIColor.lightGrayColor.

File: components/TextFields/src/MDCTextInputController.h [hunk 2]
```diff
@@ -253,6 +253,34 @@
...
+ Height of the underline when text field is first responder.
+
+ Default is underlineHeightActiveDefault.
+ */
+@property(nonatomic, assign) CGFloat underlineHeightActive;
+
+/**
+ Default value for underlineHeightActive.
+
+ Default is 2.
```
Comment (romoore) [inline components/TextFields/src/MDCTextInputController.h]: Super-nit: Follow Apple's approach of documenting default floats as floating-point literals?

    Default is 2.0.

File: components/TextFields/src/MDCTextInputController.h [hunk 3]
```diff
@@ -253,6 +253,34 @@
...
+ Height of the underline when NOT first responder.
+
+ Default is underlineHeightNormalDefault.
+ */
+@property(nonatomic, assign) CGFloat underlineHeightNormal;
+
+/**
+ Default value for underlineHeightNormal.
+
+ Default is 1.
```
Comment (romoore) [inline components/TextFields/src/MDCTextInputController.h]: Super-nit: `Default is 1.0.`

**Review by romoore — APPROVED** [review]
Minor nits, feel free to disregard.

File: components/TextFields/src/MDCTextInputController.h (hunk 2 above)
Comment (willlarche) [reply components/TextFields/src/MDCTextInputController.h]: I'll go back thru and do that for all in a nother PR.

Comment (romoore) [reply components/TextFields/src/MDCTextInputController.h]: Sounds good to me.

File: components/TextFields/src/MDCTextInputController.h (hunk 1 above)
Comment (willlarche) [reply components/TextFields/src/MDCTextInputController.h]: oh yeah! But what you didn't notice is that all these `Default is [something real]` are false! This is a protocol. I have now updated it to delete them all. (There used to be so few controllers that they weren't a lie. They are lies now.)

**Pull request merged**